source/beamline/VESPERS/VESPERSBeamline.cpp

```diff
@@ -1233,7 +1239,7 @@ QString VESPERSBeamline::details() const
 bool VESPERSBeamline::allValvesOpen() const
 {
 	for (int i = 0; i < valveSet_->count(); i++)
-		if (valveSet_->at(i)->value() == 0 || valveSet_->at(i)->value() == 2)
+		if (valveSet_->at(i)->value() == CLSExclusiveStatesControl::Open)
```

Comment:
Although this code works, but as commented above, the first element of `valveSet_` is `vvrFE1_`, which is not an instance of `CLSExclusiveStatesControl`. Therefore, it might be risky to use `CLSExclusiveStatesControl` definitions for it.